doc/releases/changelog-dev.md

```diff
@@ -218,4 +221,4 @@ Romain Moyard,
 Mudit Pandey,
 Matthew Silverman,
 Jay Soni,
-
+Yash Prabhat,
```

Comment:
The names must be ordered alphabetically with respect to your last name, so it should be earlier in the list